ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -28,6 +30,7 @@
     private final PaymentMapper paymentMapper;
     private final InterestMapper interestMapper;
     private final InterestDateMapper interestDateMapper;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "objectMapper" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1068)